training/train.py

```diff
@@ -90,6 +90,15 @@ def generate_manifest_id_dict(manifest_list, package_id_dict):
     return manifest_id_dict
 
 
+def format_dict(package_id_dict, manifest_id_dict):
+    """Format the dictionaries."""
+    format_pkg_id_dict = [{'ecosystem': 'maven',
+                          'package_list': package_id_dict}]
+    format_mnf_id_dict = [{'ecosystem': 'maven',
```

Comment:
Same as above. Can't we get rid of this array structure and directly deal with a dict instead?

Reply:
For that mites, we need to change, all the 2019-01-03 version 'trained model' structure in dev, STAGE, prod. if we are ok with that. I can do that.